Makefile.am

```diff
@@ -297,6 +301,9 @@ lcfrpginclude_HEADERS = \
 	src/generated/lcf/rpg/trooppagecondition.h \
 	src/generated/lcf/rpg/variable.h
 
+lcfthirdpartyinclude_HEADERS = \
+	src/lcf/third_party/string_view.hpp
```

Comment:
We need to make sure there are no makefiles (some video game consoles) or project builders (like Android) just with *.h wildcards.

Reply:
I can just rename it to .h if that helps with consistency